bootstrap-navbar.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Bootstrap Navbar</title>
+        <style>
+            .container {
+                max-width: 1200px;
+            }
+            .row {
+                clear: both;
+            }
+            .column {
+                width: 25x;
+                float: left;
+            }
+            ul {
+                list-style-type: none;
+                margin: 0px;
+                padding: 0px;
+                overflow: hidden;
+            } 
+            li {
+                display: inline;
+            }
+            li a {
+                display: inline;
+                background-color: indigo;
+                color: white;
+                margin: 5px;
+            }
+            button {
+                display: inline;
+                color: white;
+                background: indigo;
+            }
+        </style>
+        <script>
+            console.log("Hello world!")
+        </script>
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="column">
+                    <nav>
+                        <ul>
+                            <li><a href="#logo">Logo</a></li>
```

Comment:
give each `li` a class of `navigation-item`.
And within that class, make sure each li is `inline-block` so that all the list items are aligned horizontally